factory/declarations.py

```diff
@@ -89,6 +89,22 @@ def evaluate(self, instance, step, extra):
         return self.function(instance)
 
 
+class Transformer(LazyFunction):
+    """Transform value using given function.
+
+    Attributes:
+        transform (function): function that returns the transformed value.
+        value: value passed as the first argument to the transform function.
+    """
+
+    def __init__(self, transform, value, *args, **kwargs):
+        super(Transformer, self).__init__(transform, *args, **kwargs)
```

Comment:
As the code has since become Python 3+

```suggestion
        super().__init__(transform, *args, **kwargs)
```